use-timescale/data-tiering/about-data-tiering.md

```diff
@@ -9,45 +9,32 @@ cloud_ui:
 
-$COMPANY charges only for the storage that your data occupies in S3, regardless of whether it was compressed in $CLOUD_LONG before tiering. There are no additional expenses, such as data transfer or compute. 
+$CLOUD_LONG high-performance storage comes in two types—standard and enhanced. 
 
-## Benefits of the object storage tier
+- **Standard** (default): provides up to 16TB of storage and 16,000 IOPS. Based on [AWS EBS gp3][aws-gp3].
+- **Enhanced**: provides up to 64TB of storage and 64,000 IOPS. Based on [EBS io2][ebs-io2].
 
-The object storage tier is more than an archiving solution. It is also:
-
-*   **Cost-effective:** store high volumes of data at a lower cost.
-    You pay only for what you store, with no extra cost for queries.
-
-*   **Scalable:** scale past the restrictions imposed by storage that can be attached
-    directly to a Timescale service (currently 16 TB).
-
-*   **Online:** your data is always there and can be [queried when needed][querying-tiered-data]. 
-
-## Architecture
-
-The tiered storage backend works by periodically and asynchronously moving older chunks from the high-performance storage to the object storage. 
-There, it's stored in the Apache Parquet format, which is a compressed columnar format well-suited for S3. Within a Parquet file, a set of rows is grouped together to form a row group. Within a row group, values for a single column across multiple rows are stored together.
+[See the differences][aws-storage-types] in the underlying AWS storage. You [enable enhanced storage][enable-enhanced] as needed in $CONSOLE. 
 
-By default, tiered data is not included when querying from a Timescale service. 
-However, you can access tiered data by [enabling tiered reads][querying-tiered-data] for a query, a session, or even for all sessions. After you enable tiered reads, when you run regular SQL queries, a behind-the-scenes process transparently pulls data from wherever it's located: the standard high-performance storage tier, the object storage tier, or both.
+## Low-cost storage
 
-Various SQL optimizations limit what needs to be read from S3:
+Once you enable tiered storage, you can start moving rarely used data to the object tier. The object tier is based on AWS S3 and stores your data in the [Apache Parquet][parquet] format. Within a Parquet file, a set of rows is grouped together to form a row group. Within a row group, values for a single column across multiple rows are stored together. The original size of the data in your $SERVICE_SHORT, compressed or uncompressed, does not correspond directly to its size in S3. A compressed hypertable may even take more space in S3 than it does in $CLOUD_LONG.
```

Comment:
```suggestion
Once you enable tiered storage, $CLOUD_LONG moves rarely used data to the object tier. The object tier is based on AWS S3 and stores your data in the [Apache Parquet][parquet] format. Within a Parquet file, a set of rows is grouped together to form a row group. Within a row group, values for a single column across multiple rows are stored together. The original size of the data in your $SERVICE_SHORT, compressed or uncompressed, does not correspond directly to its size in S3. A compressed hypertable may even take more space in S3 than it does in $CLOUD_LONG.
```

Can you link to /enabling-data-tiering/#enable-tiered-storage for enable tiered storage please

Reply:
This is not technically true. Once you enable tiered storage, you need to set up policies or move data manually. Enabling it does not automatically start the move.